src/components/KInputSwitch/KInputSwitch.vue

```diff
@@ -159,6 +159,10 @@ export default defineComponent({
 .k-input-switch {
   position: relative;
 
+.has-label-left + .kong-icon {
+  left: 133px;
```

Comment:
This won't work for variable label length:
![image](https://user-images.githubusercontent.com/67973710/224425685-7103e25a-ceb2-452a-b0d6-dc8abf31b625.png)

We can't do this by hard-coding the left positioning.

Reply:
Fixed